.github/workflows/integration-build-product.yml

```diff
@@ -1477,6 +1477,29 @@ jobs:
             echo $ip
           '
 
+      - name: Remote - Detect Nvidia mismatch
+        if: ${{ inputs.with_simulation_tests }}
+        shell: bash
+        id: nvidia_check
+        run: |
+          ssh ${{ steps.infra_outputs.outputs.ssh_connect_string }} -i ~/.ssh/ci_priv_key.pem -o StrictHostKeyChecking=no '
+            set -e
+            container_id=$(docker run --gpus=all --rm nvcr.io/nvidia/cuda:11.6.2-base-ubuntu20.04 nvidia-smi 2>&1)
+            if [[ $? -ne 0 ]]; then
+              exit 1
+            fi
+          '
+          echo "msg_error=Nvidia container error detected" >> $GITHUB_OUTPUT
```

Comment:
add the url to the job for quicker jump 
Details: https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}